css/src/components/form/textarea.scss

```diff
@@ -87,3 +88,60 @@ $textarea-focus-box-shadow-size: 0 0.0625rem 0 0 !default;
 		}
 	}
 }
+
+$textarea-form-gap: 0.25em !default;
+$textarea-divider-radius: 0.5rem !default;
+
+.textarea-form {
+	display: grid;
+	gap: $textarea-form-gap;
+	padding-block: $control-padding-vertical;
+	padding-inline: $control-padding-horizontal;
+	transition: box-shadow 300ms linear;
+	border-width: $control-border-width solid transparent;
```

Comment:
Seems this was just changed from `border` to `border-width`, but the `solid transparent` is still here, so it's now an invalid declaration. As a result, the textarea has no border:

![Borderless textarea](https://github.com/user-attachments/assets/639f3d7c-e57f-4615-b0e9-a18f087dacdb)